backend/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "pixelart_town_backend",
+  "name": "backend",
```

Comment:
maybe check if the front end is pixelart town as well lol

Reply:
It doesn't